doomed_patient/doomed_patient/NPC.h

```diff
@@ -34,6 +34,40 @@ class NPC : public Character
 
 	virtual void update(LevelCell::Passable passable);
 
+
 private:
 	Directions::Direction movementDirection;
+
+	// The cell the NPC is currently occupying.
+	/*!
+	This method updates the NPCs currentCell pointer to the
+	cell that the game considers the NPC to be occupying.
+	*/
+	std::shared_ptr<LevelCell> currentCell;
```

Comment:
Use this member variable, ```currentCell``` to access the cell the NPC is currently considered to be in. It will automatically be updated in ```NPC::update()```